client/src/index.css

```diff
@@ -87,12 +114,18 @@
     width: 100%;
     text-align: center;
   }
+
 }
 
 @media (max-width: 400px) {
   .nb-photo-container {
     height: 250px !important;
```

Comment:
Avoid `!important` when possible. This breaks inheritance and makes future bugs almost impossible to fix. Try to improve specificity of the selector instead

Reply:
WIll do, was a lefetover from dealing with bootstrap defaults